test/Hydrator/MoneyHydratorTest.php

```diff
@@ -21,7 +21,7 @@ public function testHydratorExtractAsExpected()
         $object = new Money(500, new Currency('BRL'));
         $hydrator = new MoneyHydrator();
         $extracted = $hydrator->extract($object);
-        $expected = ['amount' => $object->getAmount(), 'currency' => $object->getCurrency()->getName()];
+        $expected = ['amount' => 5, 'currency' => $object->getCurrency()->getName()];
```

Comment:
The purpose of `mathiasverraes/money` is to always store and work with money represented in the smallest unit (because of rouding erros).

Yeah, of course we don't want to display it in cents for our user, that's why the `$amount / 100` is a **presentation concern**, so that you must handle it in view layer. I think a custom form element does the trick.

Thoughts? @olekhy @prolic @fabiocarneiro @gabrielsch